whipper/command/cd.py

```diff
@@ -310,6 +313,11 @@ def add_arguments(self):
                                  "{}; 0 means "
                                  "infinity.".format(DEFAULT_MAX_RETRIES),
                                  default=DEFAULT_MAX_RETRIES)
+        self.parser.add_argument('-k', '--keep-going',
```

Comment:
Maybe you could add a `default=False` argument.

Reply:
The `store_true` action automatically sets a default setting of False if the option isn't provided, the `default` argument is not needed.